pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
     "distributed",
     "fiona>=1.8.13.post1",
     "fsspec",
+    "gdal>=3.6.4",
```

Comment:
This should not be here because we don't import GDALs python bindings anywhere in the package. All GDAL related stuff is "hidden" within the builds of fiona and rasterio.